README.md

```diff
@@ -16,7 +18,28 @@ Download the dependencies for the project through [npm](https://www.npmjs.com),
 
     npm install
 
-### Resources ###
+Optionally, download some tools for building and testing the project, such as [Forever](https://github.com/foreverjs/forever) or [Jasmine](http://jasmine.github.io/). By configuring them "globally" with the ``-g``, they will be accessible from the shell.
+
+    npm install forever -g
+    npm install jasmine-node -g
```

Comment:
The other day, I had some trouble installing jasmine, but only because I forgot to add "node" to the end!